completion/available/cargo.completion.bash

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env bash
+
+# cargo (Rust package manager) completion
+
+source $HOME/.cargo/env
```

Comment:
Is it possible that this is installed anywhere else? We'd prefer if you test for this location before assuming it's there.

Reply:
I have removed this line to make it .bashrc's responsibiility to source this file since it seems generally impossible to determine the installation path.